src/specs.json

```diff
@@ -0,0 +1,51 @@
+[
```

Comment:
Love this!

Reply:
Yeah, let's iterate on this :). I think that's a great idea